ui/package.json

```diff
@@ -41,7 +41,6 @@
     "@types/lodash": "^4.14.202",
     "@types/node": "^10.12.11",
     "@types/react": "^17.0.71",
-    "@types/react-dom": "^17.0.25",
```

Comment:
`@testing-library/react` requires this, so we don't need to call it out explicitly.  Also true for `@testing-library/dom`